ui/packages/atlasmap-core/src/services/document-management.service.ts

```diff
@@ -1046,6 +1046,40 @@ export class DocumentManagementService {
     });
   }
 
+  /**
+   * Delete all user-defined documents.
+   *
+   * @returns Promise true is all documents are successfully delete, Promise
+   * false otherwise.
+   */
+  deleteAllDocuments(): Promise<boolean> {
```

Comment:
Maybe `deleteAllDocumentsAndMappings()` or `deleteMappingProject()`, and state that it will delete mappings as well as Documents in the comment?